javascript/javascript2/week1/Shortestword/shortestWord.js

```diff
@@ -0,0 +1,31 @@
+// Method 1 of finding the shortest word in an array
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium',[], 0, null, undefined, 'aa', Infinity, true];
+function findShortestWord(word) {
+let shortest = word[0];
+for(let i=0; i < word.length; i++){
+    if (typeof word[i] === 'string' &&
+        (shortest === ''|| word[i].length < shortest.length)) {
```

Comment:
this should be sufficient
```suggestion
        (word[i].length < shortest.length)) {
```

Reply:
corrected the code.